DEVELOPER.md

```diff
@@ -222,6 +222,11 @@ To pull in a merge request you should generally follow the command line instruct
 
 # Build the code and run the tests
 
+Storm has some oddities with packaging and plugin dependencies.  We build our own plugins and shade storm-core,
+but have sub-modules that depend on storm-core.  Because of this the build needs to be split into two parts.
```

Comment:
Can you please explain in a bit more detail why this split is needed, and how the change accomplishes that?